### ISSUE TITLE
Making imports more transparent

### ISSUE THREAD

**Issue by RafalSkolasinski** [topocm/topocm_content#146]
in ``init_mooc_nb.py`` there is an import
```python
from functions import *
```

Which caused me some confusion. I was looking on ``w2_majorana/nanowire.ipynb`` and it took me a while to find out where ``spectrum`` function comes from.

Maybe putting in ``init_mooc_nb.py`` 
```python
import functions
```

and in specific notebook
```python
from functions import spectrum
```

would be more transparent?

**Comment by basnijholt** [comment]
The idea is that the functions are like a blackbox and it's not necessary to know how the functions in `functions` work, neither where they are.

I think we could say in `init_mooc_nb.py`

```python
from functions import spectrum, ...
```

**Comment by RafalSkolasinski** [comment]
Making that in ``init_mooc_nb.py`` also seems like a very good idea.

**Comment by akhmerov** [comment]
We are already printing the imported names, let's also print the origin. I am against explicit imports because this would mean we need to modify all notebooks.

**Comment by basnijholt** [comment]
6b7bc9b prints:
```
Populated the namespace with:
np, matplotlib, kwant, holoviews, init_notebook, interact, display_html, plt, pf, SimpleNamespace, pprint_matrix, scientific_number, pretty_fmt_complexMoocVideo, edx_components.PreprintReference, edx_components.MoocDiscussion, edx_components.MoocCheckboxesAssessment, edx_components.MoocMultipleChoiceAssessment, edx_components.MoocPeerAssessment, edx_components.MoocSelfAssessmentspectrum, functions.hamiltonian_array, functions.h_k, functions.pauli
```

**Comment by basnijholt** [comment]
Still might not be exactly what we want.

**Comment by basnijholt** [comment]
Closed by 4f0f5f5
